### ISSUE TITLE
Add multiple account support

### ISSUE THREAD

**Issue by KlimaRitter** [Stremio/stremio-features#320]
A nice way to switch between multiple saved accounts would be awesome, specially the android tv app would become a lot more usable by this rather simple improvement!

**Comment by deltaxsingh** [comment]
Yeah I don't think they're going to. It's been requested a bunch of times, devs think it's a waste of time to add.

**Comment by Ignawesome** [comment]
I would also like to request this for Android TV

**Comment by mrtonimendes** [comment]
If there are many accounts logged in to the app, then there should be a page that opens right before the home page asking which user is using Stremio. This would make the Android TV app much muuuuuuch more useful. I would even consider this feature essential in the Android TV app
